benchmark/sphere.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>BenchMark: Sphere</title>
+</head>
+
+<body>
+
+    <meta charset="utf-8">
+    <style>
+        #sphere {
+            stroke: #444;
+            stroke-width: 2;
+        }
+
+        .polygons {
+            stroke: #444;
+        }
+
+        .sites {
+            stroke: black;
+            fill: white;
+        }
+    </style>
+
+    <h1>Benchmark: Sphere</h1>
+    <h3>d3-geo-voronoi</h3>
+    <svg width="960" height="600"></svg>
+    <h2> Suggestion: </h2>
+    <p> Enable the browsers FPS meter. </p>
+
+    <script src="https://d3js.org/d3.v5.min.js"></script>
+    <script src="https://unpkg.com/d3-delaunay@5"></script>
```

Comment:
```suggestion
    <script src="https://unpkg.com/d3@6"></script>
```

since d3@6 include d3-delaunay